scripts/ci-check_release_commit_messages.sh

```diff
@@ -7,35 +7,17 @@ git fetch origin develop
 # list all commits between HEAD and develop
 for commit in $(git rev-list origin/develop..)
 do
```

Comment:
Is the looping really needed? Because I thought the idea of `git merge-base` was to avoid this.

Reply:
Yep, you're right it's pointless now. Removed it.